uwp/src/ZWOptions.cpp

```diff
@@ -0,0 +1,141 @@
+#include "pch.h"
+#include "ZWOptions.h"
+
+using namespace OpenZWaveUWP;
+using namespace OpenZWave;
+
+//-----------------------------------------------------------------------------
+//	<ZWOptions::Create>
+//	Create the unmanaged Options singleton object
+//-----------------------------------------------------------------------------
+void ZWOptions::Create
+(
+	String^ _configPath,
+	String^	_userPath,
+	String^	_commandLine
```

Comment:
I would suggest hardcoding these, or at least provide a simpler overload. The nuget package would always deploy the configs in the same folder, the userPath would always be `Windows::Storage::ApplicationData::Current::LocalFolder->Path`.